orm.go

```diff
@@ -112,26 +112,44 @@ func (s *Schema) diff(other *Schema) *Schema {
 	return result
 }
 
+type serializedQuery struct {
+	Query   string
+	HasKeys bool
+}
+
+func NewSerializedQuery(query string, hasKeys bool) serializedQuery {
```

Comment:
this function should be unexported as well, I think

Reply:
Unexported.